data/test_config/config_defaults.yml

```diff
@@ -24,3 +24,16 @@ pipeline: []
 #     ambiguity_threshold: 0.1
 
 data:
+policies:
+# # No configuration for policies was provided. The following default policies were used to train your model.
+# # If you'd like to customize them, uncomment and adjust the policies.
+# # See https://rasa.com/docs/rasa/policies for more information.
+#   - name: MemoizationPolicy
+#   - name: RulePolicy
+#   - name: UnexpecTEDIntentPolicy
+#     max_history: 5
+#     epochs: 100
+#   - name: TEDPolicy
+#     max_history: 5
+#     epochs: 100
+#     constrain_similarities: true
```

Comment:
is this change actually needed?

Reply:
I think it happens around here:
https://github.com/RasaHQ/rasa/blob/d13b153574acb1a70dd4c170a8bc23b514e48a63/rasa/shared/importers/autoconfig.py#L67

Reply:
Mhm, I see your point. We should probably change the tests to not write there instead 🙈 Fine to keep it as it is 👍🏻

Reply:
yep, but in my opinion the tests should e.g. copy the config before doing this.

Reply:
Yes, that would be another way to do it; in general, I wonder if just writing over an existing file is really what we want. I'm assuming that in most cases - although not here - this would happen in a more interactive context anyways, where it might also make sense, for instance, to ask the user if they want these default comments to be added.

Reply:
In my opinion fine in _this_ case as we're just adding a comment and not actually changing something 🤔 Agree with you in general though 👍🏻